Cargo.toml

```diff
@@ -14,8 +14,9 @@ crate-type = ["cdylib"]
 pyo3-build-config = "0.19.1"
 
 [dependencies]
+base64 = "0.21"
 chrono = "0.4.23"
-tantivy = "0.20.1"
```

Comment:
Can you change the package version to 0.21.0 above as well

Reply:
What do you mean? The change to 0.21.0 is below

Reply:
He means the version number of tantivy-py itself I think.

Reply:
and also, not just `tantivy-py`, but our fork of tantivy-py which might need it's own sub-patch version number.

For example
- if tatnivy is 0.21.0
- tantivy-py could be 0.21.0.0
- our fork of tantivy-py something like 0.21.0.0.post1 or similar.

I am not sure.

Reply:
Ah ok. Totally missed that. I'll make the change upstream first. It's only the tag and release that needs to be that scheme right?

Reply:
It's a good question. I am not sure whether the version string in the upstream tantivy-py `Cargo.toml` needs to be changed, since we never actually install the "crate", we install the Python wheel.

However, I guess it is theoretically possible that someone else might want to install the tantivy-py _crate_, perhaps to extend it as we have done with our Kapiche fork.  I feel like the version string in `Cargo.toml` should probably align with the tag and the release, but honestly I am not sure.